lib/foreman/renderer/scope/macros/host_template.rb

```diff
@@ -90,7 +90,7 @@ def host_param_false?(name, default_value = false)
             if host.params.has_key?(name)
               Foreman::Cast.to_bool(host.params[name]) == false
             else
-              default_value
+              default_value == false
             end
```

Comment:
I kind of wonder about some equivalents:
```ruby
Foreman::Cast.to_bool(host.params.fetch(name, default_value)) == false
```

Or:
```ruby
value = host.params.fetch(name, default_value)
Foreman::Cast.to_bool(value) == false
```

As you already noted, this makes it clearer what the intent is.